experiments/integrated/ozark/ozark.jl

```diff
@@ -635,130 +574,168 @@ swp = [
-    ylim = [-5, 7.5],
-    size = (1500, 400),
-)
-Plots.plot!(plt1, daily, stem_leaf_flux, label = "")
-Plots.plot!(plt1, daily, root_stem_flux, label = "")
-Plots.plot(plt2, plt1, layout = (2, 1))
+    label="",
+    xlabel="Day of year",
+    margin=10Plots.mm,
+    xlim=[minimum(daily), minimum(daily) + 30],
+    ylim=[-5, 7.5],
+    size=(1500, 400),
+)
+Plots.plot!(plt1, daily, stem_leaf_flux, label="")
+Plots.plot!(plt1, daily, root_stem_flux, label="")
+Plots.plot(plt2, plt1, layout=(2, 1))
 Plots.savefig(joinpath(savedir, "water_fluxes.png"))
 
+# Soil Temperature
+
 # Current resolution is 3.333 cm per layer, our nodes are at the center of the
 # layers. The second layer is ~ 5cm
 soil_T_sfc = [parent(sv.saveval[k].soil.T)[end] for k in 1:length(sol.t)]
-soil_T_5 = [parent(sv.saveval[k].soil.T)[end - 1] for k in 1:length(sol.t)]
+soil_T_5 = [parent(sv.saveval[k].soil.T)[end-1] for k in 1:length(sol.t)]
 
 plt2 = Plots.plot()
 Plots.plot!(
     plt2,
     daily,
     soil_T_sfc,
-    color = "blue",
-    label = "",
-    xlim = [minimum(daily), minimum(daily) + 30],
-    ylim = [260, 320],
-    xlabel = "Day of year",
-    margins = 10Plots.mm,
-    size = (1500, 400),
+    color="blue",
+    label="",
+    xlim=[minimum(daily), minimum(daily) + 30],
+    ylim=[260, 320],
+    xlabel="Day of year",
+    margins=10Plots.mm,
+    size=(1500, 400),
 )
 
-plot!(plt2, daily, soil_T_5, color = "purple", label = "")
-Plots.plot!(plt2, seconds ./ 3600 ./ 24, TA, label = "", color = "green")
-Plots.plot!(plt2, seconds ./ 3600 ./ 24, TS, label = "", color = "red")
+plot!(plt2, daily, soil_T_5, color="purple", label="")
+Plots.plot!(plt2, seconds ./ 3600 ./ 24, TA, label="", color="green")
+Plots.plot!(plt2, seconds ./ 3600 ./ 24, TS, label="", color="red")
 
 
 plt3 = Plots.plot()
 Plots.plot!(
     plt3,
     daily,
     soil_T_sfc,
-    color = "blue",
-    label = "T_sfc",
-    xlim = [minimum(daily), maximum(daily)],
-    ylim = [260, 320],
-    title = "Soil Temperature [K]",
+    color="blue",
+    label="T_sfc",
+    xlim=[minimum(daily), maximum(daily)],
+    ylim=[260, 320],
+    title="Soil Temperature [K]",
 )
 
-plot!(plt3, daily, soil_T_5, color = "purple", label = "T_soil, 5cm")
+plot!(plt3, daily, soil_T_5, color="purple", label="T_soil, 5cm")
 Plots.plot!(
     plt3,
     seconds ./ 3600 ./ 24,
     TA,
-    label = "T_air (data)",
-    color = "green",
+    label="T_air (data)",
+    color="green",
 )
 
 Plots.plot!(
     plt3,
     seconds ./ 3600 ./ 24,
     TS,
-    label = "T_soil (data)",
-    color = "red",
-    legend = :bottomright,
+    label="T_soil (data)",
+    color="red",
+    legend=:bottomright,
 )
 
-Plots.plot!(plt3, legend = :bottomleft)
-Plots.plot(plt3, plt2, layout = (2, 1))
+Plots.plot!(plt3, legend=:bottomleft)
+Plots.plot(plt3, plt2, layout=(2, 1))
 Plots.savefig(joinpath(savedir, "soil_temperature.png"))
 
-
-
-
+# Run script with comand line argument "save" to save model output to CSV
```

Comment:
this looks great! how would you feel about a future PR which moves this into src?

@juliasloan25